pkg/kafka/auth/config.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auth
+
+import (
+	"log"
+	"strings"
+
+	"github.com/Shopify/sarama"
+	"github.com/spf13/viper"
+)
+
+const none = "none"
+const kerberos = "kerberos"
```

Comment:
nit: I would define this as array and pass to flag description:

* to avoid duplication in constants
* this way we don't forget to update help message if new auth is added